### ISSUE TITLE
Implement SSM Parameter Support

### ISSUE THREAD

**Issue by captind** [KCOM-Enterprise/cfn-square#8]
We should implement SSM parameter support by allowing an environment substitution via a command line variable. This will allow us to look up cloud formation parameters from an SSM parameter store on a per environment basis.

**Comment by jrudge-kcom** [comment]
+1

**Comment by steveparker-1984** [comment]
I could potentially do with two features here. The first is as @captind describes, the other is SSM substitution in the template url. If for example we followed the packer style you might have something like this:

```
stSomeStack:
    template-url: s3://{{ssm `templatesbucket`}}/resources/templates/some.template
    timeout: 300
    parameters:
        pSomeSSMParam: /'{{user `env`}}'/SomeParam
```
Called with: cf sync -y stacks.yml -var 'env=dev'

Thoughts?

**Comment by rayoogmy** [comment]
If we add the key=value as well as ssm then that will give us a lot of customisation already. So yes, I think they are all useful features to have. We should split ssm into another feature request though. 

I will have a look at both.

**Comment by nathanreynolds-kcom** [comment]
Looks to have been implemented by #17
